common/defs/bitcoin/zcore.json

```diff
@@ -0,0 +1,43 @@
+{
+  "coin_name": "ZCore",
+  "coin_shortcut": "ZCR",
+  "coin_label": "ZCore",
+  "website": "https://zcore.cash",
+  "github": "https://github.com/zcore-coin/zcore-2.0",
+  "maintainer": "Erick Costa <erick@zcore.cash>",
+  "curve_name": "secp256k1",
+  "address_type": 142,
+  "address_type_p2sh": 0,
```

Comment:
Are you sure this is the correct address_type?

We can't accept a coin which has address version that collides with Bitcoin (that is addresses start with `1` or `3`).

Reply:
It's correct, 142 starts with `z` only.
I've tested the firmware and it generates addresses with `z`.

https://en.bitcoin.it/wiki/List_of_address_prefixes

Reply:
I am talking about `0` for address_type_p2sh.
It is indeed 0 here:
https://github.com/zcore-coin/zcore-2.0/blob/master/src/chainparams.cpp#L176
but that is unacceptable value. It means that multisig addresses of ZCore will start with `1` resembling the Bitcoin addresses.

Please change this value in zcore and here and then we will merge.